### PR TITLE
Fixes Logout Crash

### LINKED ISSUE #3131: Crash when signing out while blog is syncing all the things.
Edge case, but to reproduce:
From the My Sites tab, tap in to view a site.  This will kick off a batch of requests to sync data. 
Switch to the Me tab and sign out. 
Crash. 

Probably a bit of a race condition to signout and delete the blog records before the sync completes.

### PR DESCRIPTION
**Pull request by jleandroperez**
Fixes #3131

Note: Please, merge [this pr](https://github.com/wordpress-mobile/WordPressApi/pull/26) first + release a new *WordPressApi* pod.

Thanks!

/cc @sendhil

### CODE REVIEW

**Comment by jleandroperez** [comment]
@sendhil I've re-mapped the WordPressApi POD to Main (it was mapped to a tag that didn't contain the latest changes). Plus, merge conflicts were solved.

May i bother you with a quick review?

Thanks!

File: Podfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ pod 'UIDeviceIdentifier', '~> 0.1'
 pod 'SVProgressHUD', :git => 'https://github.com/TransitApp/SVProgressHUD.git', :commit => '2ac46ae2f2dd37db153ce1daea73b1273ac2bba3'
 pod 'AMPopTip', '~> 0.7'
 pod 'wpxmlrpc', '~> 0.7'
-pod 'WordPressApi', :git => 'https://github.com/wordpress-mobile/WordPressApi.git', :tag => '0.2.1'
+pod 'WordPressApi', :git => 'https://github.com/wordpress-mobile/WordPressApi.git', :branch => 'master'
```
Comment (sendhil) [inline Podfile]: Why not map it to a tag?

**Comment by sendhil** [comment]
@jleandroperez this fix looks good - but would you mind making sure we map `WordPressApi` to a tag instead of master? Once that's done :shipit:

**Comment by jleandroperez** [comment]
@sendhil will do sir, thank you very much!

**Pull request merged**